.editorconfig

```diff
@@ -0,0 +1,16 @@
+# archlinux-proaudio editorconfig
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[PKGBUILD]
+indent_style = spaces
+indent_size = 2
+max_line_length = 100
+binary_next_line = true
+space_redirects = true
+keep_padding = true
```

Comment:
What are those three settings? They are not in the specification, AFAICS.

Reply:
Hmm, you're right, they're not in the spec. I found them in the [shfmt manpage](https://github.com/mvdan/sh/blob/master/cmd/shfmt/shfmt.1.scd#examples) where an analogous `.editorconfig` is provided 🤔 I'll just remove it to avoid confusion, at least with vim-editorconfig these didn't have any effect.